components/SwipeRow.js

```diff
@@ -458,7 +458,7 @@ class SwipeRow extends Component {
         return (
             <TouchableOpacity
                 activeOpacity={1}
-                onPress={this.combinedOnPress}
+                onPress={this.combinedOnPress()}
```

Comment:
Hi @kubilaykiymaci what problem are you trying to solve here?

The code you wrote will actually run the onPress function at the time the component is rendered, not at the time the component is pressed

Reply:
Hi @jemise111, you're so right and it was my mistake :(

I checked it again, onPress works great.

Reply:
👍